src/customize-pop/gnome-tweaks-extensions.md

```diff
@@ -0,0 +1,202 @@
+
+#### Lock Screen Options
+
+| Option | Function |
+|--------|----------|
+| Image | Select an image that will display at the lock screen. |
+| Adjustment | Manipulate the appearance of the lock screen background. Options include: <ul><li>Zoom</li><li>Centered</li><li>Scaled</li><li>Spanned</li><li>Stretched</li><li>Wallpaper |
+
+### Fonts
+
+`Fonts` settings allow you to specify the default fonts used throughout the operating system.
+
+![Fonts Tweaks](/images/gnome-tweaks-extensions/fonts-tweaks.png)
+
+| Option | Function |
+|--------|----------|
+| Interface Text | Choose the font used for displayed text inside of a running application. |
+| Document Text | Choose the font used when creating a new text document. Applications like Libreoffice may override this selection. |
+| Monospace Text | Choose the font used in a Terminal session. This may also include coding applications and basic text editors. |
+| Legacy Window Titles | Choose the font used to display information in the top bar of running application windows. This is usually the name of the application. |
+| Hinting | Enable text hinting, which improves text readability for low-resolution displays. |
+| Antialiasing | Enable antialiasing, which improves text visibility when high-resolution text is displayed on a low-resolution screen. |
+| Scaling Factor | Enable text scaling, which increases font size for HiDPI displays while retaining text sharpness. |
+
+### Keyboard & Mouse
+
+`Keyboard & Mouse` settings extend customization of keyboard and mouse input behavior.
+
+![Keyboard Mouse Tweaks](/images/gnome-tweaks-extensions/keyboard-mouse-tweaks.png)
+
+#### Keyboard
+
+| Option | Function |
+|--------|----------|
+| Show Extended Input Sources | Display additional number of input sources found in the Settings application. |
+| Emacs Input | Apply Emacs keybindings throughout the operating system. |
+| Overview Shortcut | Choose whether left or right `SUPER` keys start the Launcher. |
+
+#### Mouse
+
+| Option | Function |
+|--------|----------|
+| Acceleration Profile | Choose how mouse movement speed adapts to user gestures. |
+| Pointer Location | Pressing the `Ctrl` key creates a visual notification at the mouse pointer location. |
+| Middle Click Paste | Clicking the middle mouse button pastes content from the clipboard. |
+| (Touchpad) Disable While Typing | The touchpad is disabled when using the keyboard. |
+| (Touchpad) Mouse Click Emulation | Select touchpad input that will be interpreted as input from a standard mouse: <ul><li>Fingers: Click with two fingers on the touch pad to input right-click, and three fingers input a middle-click.</li><li>Area: Click the bottom right of the touch pad to input right-click, and the bottom middle for a middle-click.</li><li>Disabled: The touch pad input will not be interpreted as right, middle, or left-click actions. |
```

Comment:
"Select touchpad input that will be interpreted as input from a standard mouse:" sounds complicated. Consider trying to reword this to be a little less techincal, i.e. not using the word "interpreted." Out of everything in GNOME Tweaks, this is one of the common ones I would expect a non-power-user to actually need/want. For example:

> Choose how different click types are performed on the touchpad.

There are some great illustrations of at least the area method for libinput that you can find online. Making or using illustrations to demonstrate what the typical areas are (since they are absolutely not configurable under the hood) might be useful.

Reply:
Updated with your suggested wording, I'll add illustrations when this section is split out.

Reply:
Illustrations have been added.